src/index.ts

```diff
@@ -1,12 +1,12 @@
 import { Worker, MessageChannel, MessagePort, receiveMessageOnPort } from 'worker_threads';
 import { once } from 'events';
-import EventEmitterAsyncResource from 'eventemitter-asyncresource';
+import EventEmitterAsyncResource = require('eventemitter-asyncresource');
```

Comment:
I believe we should group them and companion them with a note of why we are importing them in this manner.

Reply:
Can do. But note that if we set `esModuleInterop` to `false`, it's a TS error to not do this